config/locales/es.yml

```diff
@@ -2465,103 +2465,103 @@ es:
-          change_in_filing_status: Cambio en el estado civil de presentación de la declaración federal a estatal
+          change_in_filing_status: Cambio en el estado civil tributario de la declaración federal a estatal
           dependents_not_claimed_fed_return: Agregar dependientes no reclamados en la declaración federal
-          earned_on_reservation: Ingresos obtenidos en una reserva por un Indio Americano
-          education_contribution_credit: Crédito de Aportes de Entidades Educativas
-          id_college_savings_program: Resta del programa de ahorro para la universidad de Idaho
-          id_youth_rehab_contributions: Crédito para contribuciones a centros de rehabilitación y jóvenes de Idaho
+          earned_on_reservation: Ingresos Obtenidos en una Reserva por un Indígena Estadounidense 
+          education_contribution_credit: Crédito de Aportes a Entidades Educativas
+          id_college_savings_program: Resta del Programa de Idaho Para el Ahorro Universitario
+          id_youth_rehab_contributions: Crédito de Idaho por Contribuciones a Centros de Rehabilitación y Jóvenes
           itemized_deductions: Deducciones detalladas
-          long_term_care_insurance_subtraction: Resta del seguro de atención a largo plazo
-          maintaining_elderly_disabled_credit: Crédito para mantener un hogar para personas mayores o discapacitadas
+          long_term_care_insurance_subtraction: Resta del Seguro de Atención a Largo Plazo
+          maintaining_elderly_disabled_credit: Crédito por Mantener un Hogar para Personas Mayores o Discapacitadas
           voluntary_donations: Todas las donaciones voluntarias
         id_supported:
-          child_care_deduction: Resta de cuidado de niños y dependientes de Idaho
+          child_care_deduction: Resta de cuidado de niños y dependientes
           health_insurance_premiums_subtraction: Resta de primas de seguro médico
-          id_child_tax_credit: Crédito Fiscal para Hijos de Idaho
-          id_grocery_credit: Crédito por Compras de Alimentos de Idaho
+          id_child_tax_credit: Crédito Tributario de Idaho por Hijos
+          id_grocery_credit: Crédito de Idaho por Compras de Alimentos
           interest_from_obligations: Resta de intereses de obligaciones del gobierno de EE. UU.
           social_security_retirement_deduction: Deducciones de Seguridad Social y Beneficios de Jubilación
         md_reveals:
           body1_html: |
             <ul>
               <li>
-                Sustracciones:
+                Resta:
                 <ul>
                   <li>Suministros para el aula para maestros de grados K a 12</li>
                   <li>Cuotas sindicales</li>
-                  <li>Pagos de un sistema de pensiones a bomberos y policías por lesiones o discapacidades relacionadas con el trabajo</li>
+                  <li>Pagos de un sistema de pensiones a bomberos y policías por lesiones o discapacidades relacionadas con el trabajo </li>
```

Comment:
[dust] odd we had to add a white space -- must be trying to copy english copy, not too worried about this tho (same for the list item below "Exclusión")